components/ui-shell/index.scss

```diff
@@ -108,15 +131,13 @@ $prefix: "bx";
   }
 }
 
-:host(#{$q}-header-menu-item) {
-  /* stylelint-disable-next-line scss/at-extend-no-missing-placeholder */
-  @extend .#{$prefix}--header__menu-item;
-
+#{$prefix}-header-menu-item {
   height: 3rem;
   background-color: var(--cds-ui-01);
   color: var(--cds-text-secondary);
 
-  &::part(link) {
+  &::part(link),
```

Comment:
Also for the focus state in the dropdown
<img width="504" alt="Screen Shot 2022-09-09 at 7 40 05 AM" src="https://user-images.githubusercontent.com/8097306/189376314-97d08fd5-fac4-49cf-891f-9d6e521d2c8e.png">


```suggestion
  &::part(link),
  &::part(link):focus,
```